src/analysis/specific_limits.lean

```diff
@@ -218,41 +249,77 @@ end
 
 section edist_le_geometric
 
-variables [emetric_space α] (r C : nnreal) (hr : r < 1) {f : ℕ → α}
+variables [emetric_space α] (r C : ennreal) (hr : r < 1) (hC : C ≠ ⊤) {f : ℕ → α}
   (hu : ∀n, edist (f n) (f (n+1)) ≤ C * r^n)
 
-include hr hu
+include hr hC hu
```

Comment:
Since `hr` and `hC` are only used in one lemma, it would make sense to remove them of the list of `variables`, and only put them in the relevant lemma (avoiding the `include / omit` incantations).

Reply:
My idea was to have them right after `r` and `C`, and before `hu` in the argument list, same below.

Reply:
OK, it makes sense.